docs/docsite/rst/os_guide/intro_zos.rst

```diff
@@ -0,0 +1,247 @@
+
+* ansible.builtin.script
+    The built in script module copies a local script file to a temp file on the remote target and runs it.
+    The issue that z/OS UNIX System Services targets run into is that when the underlying z/OS shell attempts to read
+    the script file, since the file does not get tagged as UTF-8 text, the shell assumes that the file is encoded in EBCDIC,
+    and fails to correctly read or run the script.
+    One work-around is to manually copy local files to managed nodes (``ansible.builtin.copy`` ) and convert or tag files (with the ``ansible.builtin.command`` module).
+    With this work-around, some of the conveniences of the script module are lost, such as automatically cleaning up the script file once it's run,
+    but it is trivial to perform those steps as additional playbook tasks.
+
+    .. code-block:: yaml
+
+        - name: Copy local script file to remote node.
+          ansible.builtin.copy:
+            src: "{{ playbook_dir }}/local/scripts/sample.sh"
+            dest: /u/ibmuser/scripts/
+
+        - name: Tag remote script file.
+          ansible.builtin.command: "chtag -tc ISO8859-1 /u/ibmuser/scripts/sample.sh"
+
+        - name: Run script.
+          ansible.builtin.command: "/u/ibmuser/scripts/sample.sh"
+
+    Another work-around is to store local script files in EBCDIC.
+    They may be unreadable on the controller, but they will copy correctly to z/OS UNIX System Services targets in EBCDIC,
+    and the script will run. This approach takes advantage of the built-in conveniences of the script module,
+    but managing unreadable EBCDIC files locally makes maintaining those script files more difficult.
+
+.. _configure_zos_remote_environment:
+
+Configure the Remote Environment
+--------------------------------
+
+Certain Language Environment (LE) configurations enable automatic encoding conversion and automatic file tagging functionality required by Python on z/OS systems (`IBM Open Enterprise SDK for Python <https://www.ibm.com/products/open-enterprise-python-zos>`__ ).
+
+Include the following configurations when setting the remote environment for any z/OS managed nodes:
+
+.. code-block:: yaml
+
+    _BPXK_AUTOCVT: "ON"
+    _CEE_RUNOPTS: "FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"
+
+    _TAG_REDIR_ERR: "txt"
+    _TAG_REDIR_IN: "txt"
+    _TAG_REDIR_OUT: "txt"
+
+
+Ansible can be configured with remote environment variables in these options:
+
+
+    * inventory - inventory.yml, group_vars/all.yml, or host_vars/all.yml
+    * playbook - ``environment`` variable at top of playbook.
+    * block or task - ``environment`` key word.
+
+For more details, see :ref:`playbooks_environment`.
+
+Configure the Remote Python Interpreter
+---------------------------------------
+
+Ansible requires a Python interpreter to run most modules on the remote host, and it checks for python at the 'default' path ``/usr/bin/python``.
```

Comment:
It should be either (if you mean Python, the programming language)
```suggestion
Ansible requires a Python interpreter to run most modules on the remote host, and it checks for Python at the 'default' path ``/usr/bin/python``.
```
or (if you mean `python`, the CLI program)
```suggestion
Ansible requires a Python interpreter to run most modules on the remote host, and it checks for ``python`` at the 'default' path ``/usr/bin/python``.
```
Same below in several other instances.